models/DataObject/ClassDefinition/Data/Numeric.php

```diff
@@ -561,7 +561,7 @@ public function isDiffChangeAllowed($object, $params = [])
      */
     public function isEmpty($data)
     {
-        return strlen($data) < 1;
+        return empty($data) || strlen($data) < 1;
```

Comment:
```suggestion
        return is_numeric($data);
```
This should do the trick in this case.